package.json

```diff
@@ -65,26 +64,27 @@
     "e2e:dev:watch": "start-test develop ':3000/status/ping|8000' cypress:dev:watch",
     "e2e:prd:run": "npm run build && start-test ':3000/status/ping|8000' cypress:dev:run",
     "e2e:prd:watch": "npm run build && start-test ':3000/status/ping|8000' cypress:dev:watch",
-    "ensure-env": "cd ./tools/ && tsc --incremental && cross-env DEBUG=fcc:* node ./scripts/build/ensure-env.js",
+    "ensure-env": "cross-env DEBUG=fcc:* ts-node ./tools/scripts/build/ensure-env.ts",
     "format": "run-s format:eslint format:prettier",
     "format:eslint": "eslint . --fix",
     "format:prettier": "prettier --write .",
     "hooks:install": "node node_modules/husky/husky.js install",
     "hooks:uninstall": "node node_modules/husky/husky.js uninstall",
-    "lint": "npm-run-all ensure-env -p lint:*",
+    "lint": "npm-run-all prepare-config -p lint:*",
     "lint:challenges": "cd ./curriculum && npm run lint",
     "lint:js": "eslint --max-warnings 0 .",
-    "lint:ts": "tsc && tsc -p tools/ui-components && tsc -p tools",
+    "lint:ts": "tsc & tsc -p config & tsc -p tools/ui-components",
     "lint:prettier": "prettier --list-different .",
+    "prepare-config": "tsc -p config && npm run ensure-env",
```

Comment:
```suggestion
    "create:config": "tsc -p config && npm run ensure-env",
```

Reply:
I should have thought of that. Thanks @raisedadead